tests/python/utils.py

```diff
@@ -0,0 +1,41 @@
+from mlp.utils import check_path
+
+def check_motion(testCase, planner, contains_all_data = True):
```

Comment:
style: according to [pep8](https://www.python.org/dev/peps/pep-0008/#other-recommendations), `=`  should not be surrounded by spaces in function prototypes.